5.5/examples/replica/persistent_nfs_volume.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
rather than provide this (because do we really want to provide this in every sample that needs a persistent volume?) i'd prefer to see us link to good documentation about how to create persistent volumes.